src/Nethermind/Nethermind.JsonRpc/JsonRpcProcessor.cs

```diff
@@ -79,7 +80,7 @@ private void BuildTraceJsonSerializer()
             _traceSerializer = Newtonsoft.Json.JsonSerializer.Create(jsonSettings);
         }
 
-        private (JsonRpcRequest Model, List<JsonRpcRequest> Collection) DeserializeObjectOrArray(string json)
+        private (JsonRpcRequest Model, List<JsonRpcRequest> Collection) DeserializeObjectOrArray1(string json)
```

Comment:
DeserializeObjectOrArray1 hmmm strange method name for me

Reply:
Yep, should be removed.